core/chainlink.Dockerfile

```diff
@@ -51,7 +51,7 @@ FROM ubuntu:20.04
 
 ARG CHAINLINK_USER=root
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y ca-certificates wget gnupg lsb-release
+RUN apt-get update && apt-get install -y ca-certificates wget gnupg lsb-release curl
```

Comment:
Can you swap the `wget` usage for `curl` on L57 and drop it? No point in keeping both, it just grows the image size

Reply:
Should be just `wget --quiet -O -` -> `curl`